src/Equinor.ProCoSys.IPO.Email/EmailService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Net;
+using System.Net.Mail;
+using System.Threading;
+using System.Threading.Tasks;
+using Equinor.ProCoSys.IPO.Email.Settings;
+using Microsoft.Extensions.Options;
+
+namespace Equinor.ProCoSys.IPO.Email
+{
+    public class EmailService : IEmailService
```

Comment:
can't see any unit tests. Add new unit-test project below Tests folder

Reply:
Have we discussed that this don't deserve any unit test? Was it that the SmtpClient is hard to Mock?

Reply:
The conclusion was that adding unit tests to SmtpClient wasa more likely to introduce errors on our side. Ref. Henning